utils.py

```diff
@@ -123,3 +123,7 @@ def weather(cmd):
 def ohno(cmd):
 	url = 'https://www.raylu.net/f/ohno/ohno%03d.png' % random.randint(1, 294)
 	cmd.reply('', {'image': {'url': url}})
+
+def ohyes(cmd):
+	url = 'https://www.raylu.net/f/ohyea/ohyea%03d.gif' % random.randint(1, 6)
```

Comment:
I don't think we'll ever hit 100 images
```suggestion
	url = 'https://www.raylu.net/f/ohyea/ohyea%02d.gif' % random.randint(1, 6)
```

where'd you get the ones you DMed me from anyway?

Reply:
internet and cutting youtube vids to gifs. Works for me. I might go full adhd one day and fill to 99